controllers/controllers.js

```diff
@@ -40,5 +40,22 @@ const getArticles = (req, res, next) => {
 }
 
 
+const postCommentToArticle = (req, res, next) => {
+    const { username, body} = req.body
+    const {article_id} = req.params
 
-module.exports = { getTopics, getEndpoints, getArticleById, getArticles }
\ No newline at end of file
+    if(!username || !body || typeof username !== 'string' || typeof body !== 'string' || Object.keys(req.body).length !== 2 ) {
```

Comment:
Good job working on the feedback to remove the unnecessary function! however, we don't need to do this check here. PSQL can deal with this as it will send back a PSQL error (e.g. if we try to insert a comment with an invalid or non existent username, it will trigger a PSQL error due to it being an invalid foreign key). Try removing this and see what happens and what errors you get, then work from there :)